server/src/internalClusterTest/java/org/opensearch/explain/ExplainActionIT.java

```diff
@@ -267,9 +271,17 @@ public void testExplainDateRangeInQueryString() {
 
         refresh();
 
-        ExplainResponse explainResponse = client().prepareExplain("test", "1").setQuery(queryStringQuery("past:[now-2M/d TO now/d]")).get();
-        assertThat(explainResponse.isExists(), equalTo(true));
-        assertThat(explainResponse.isMatch(), equalTo(true));
+        try {
+            assertBusy(() -> {
+                ExplainResponse explainResponse = client().prepareExplain("test", "1")
+                    .setQuery(queryStringQuery("past:[now-2M/d TO now/d]"))
+                    .get();
+                assertThat(explainResponse.isExists(), equalTo(true));
+                assertThat(explainResponse.isMatch(), equalTo(true));
+            });
+        } catch (Exception e) {
+            throw new RuntimeException(e);
```

Comment:
I think we can throw an Opensearch exception?

Reply:
Sure, I will update this.

Reply:
You can add throws clause to method definition similar to what we have done for other methods.